Error when trying to download with links in a file

Hi, after the update 2.1.0 when i try to download videos from a file (".txt"). It gives me this error:

[ERROR] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
    at validateString (internal/validators.js:120:11)
    at Object.join (path.js:375:7)
    at D:\destreamer-master\build\src\VideoUtils.js:78:64
    at Array.forEach (<anonymous>)
    at Object.createUniquePath (D:\destreamer-master\build\src\VideoUtils.js:75:12)
    at downloadVideo (D:\destreamer-master\build\src\destreamer.js:101:33)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)

how can i fix this?

I got the same issue. Besides, when chromium get opened, it gets crashed in the next 10-15 seconds and it returns the following error.

`[ERROR] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received undefined
at validateString (internal/validators.js:120:11)
at Object.join (path.js:375:7)
at D:\destreamer-master\build\src\VideoUtils.js:78:64
at Array.forEach ()
at Object.createUniquePath (D:\destreamer-master\build\src\VideoUtils.js:75:12)
at downloadVideo (D:\destreamer-master\build\src\destreamer.js:101:33)
at processTicksAndRejections (internal/process/task_queues.js:97:5)`

--- a/src/Utils.ts
+++ b/src/Utils.ts
@@ -127,6 +127,7 @@ export async function parseInputFile(inputFile: string, defaultOutDir: string,
         if (foundUrl) {
             outDirList.push(...Array(guidList.length - outDirList.length)
                 .fill(defaultOutDir));
+            foundUrl = false;
         }
 
         const guids: Array<string> | null = await extractGuids(line, apiClient);
@@ -138,6 +139,12 @@ export async function parseInputFile(inputFile: string, defaultOutDir: string,
         else {
             logger.warn(`Invalid url at line ${i + 1}, skipping..`);
         }
+    }
+
+    // if foundUrl is still true after the loop we have some url without an outDir
+    if (foundUrl) {
+        outDirList.push(...Array(guidList.length - outDirList.length)
+            .fill(defaultOutDir));
     }
 
     return [guidList, outDirList];

